.github/actions/create-release-v1/action.yml

```diff
@@ -0,0 +1,132 @@
+name: Create release
+description: A GitHub Action to create a production release
+
+inputs:
+  token:
+    description: The GitHub token to use. Defaults to GITHUB_TOKEN
+    default: ${{ github.token }}
+  base:
+    description: The base branch to merge into
+    required: true
+  migrations-dir:
+    description: The directory containing the database migrations e.g. src/database/migrations
+  env-file-path:
+    description: The path to the `.env` file e.g. src/service/sample.env
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        # Fetch all history
+        fetch-depth: 0
+
+    - name: Is release in progress
+      uses: dequelabs/axe-api-team-public/.github/actions/is-release-in-progress-v1@main
+      id: is_release_in_progress
+      with:
+        github-token: ${{ inputs.token }}
+
+    - name: Cancel release if a release is already in progress
+      shell: bash
+      if: ${{ steps.is_release_in_progress.outputs.in_progress == 'true' }}
+      run: |
+        echo "A release is already in progress. Cancelling this release."
+        gh run cancel ${{ github.run_id }}
+        gh run watch ${{ github.run_id }}
+
+    - name: Get release version
+      id: get_release_version
+      shell: bash
+      run: |
+        git checkout release
+
+        # Get the version number from the prepare_release shell script and set it as an output
+        # Monorepos have a lerna.json file that contains the updated version number
+        # Non-monorepos, or non-npm repos, have a package.json file that contains the updated version number
+        if [ -f "lerna.json" ]; then
+          echo "RELEASE_VERSION=$(cat lerna.json | jq -r '.version')" >> $GITHUB_OUTPUT
+        else
+          echo "RELEASE_VERSION=$(cat package.json | jq -r '.version')" >> $GITHUB_OUTPUT
+        fi
+
+    - name: Get CHANGELOG changes
+      shell: bash
+      run: |
+        echo "## Changelog Additions:" >> /tmp/release-notes.txt
+        echo "" >> /tmp/release-notes.txt
+
+        new_changelog_entries=$(git show origin/${{ inputs.base }}...origin/release --no-color --no-prefix --output-indicator-new=! CHANGELOG.md | egrep '^!' | awk -F'^[!]' '{print $2}' | sed -e 's/\n/$0A/g')
+
+        echo "$new_changelog_entries" >> /tmp/release-notes.txt
+        echo "" >> /tmp/release-notes.txt
+
+    - name: Find new migration files
```

Comment:
The migration step skips in the bash script while the env file skips using GHA `if` clause. Should make it consistent.